lib/broccoli/ember-app.js

```diff
@@ -38,6 +38,8 @@ var merge         = require('lodash/object/merge');
 var omit          = require('lodash/object/omit');
 var Funnel        = require('broccoli-funnel');
 
+var deprecate     = require('../utilities/deprecate');
```

Comment:
NINJA UPDATE: Disregard my above lies. Didn't know it existed on `this.project`.